tests/results_benchmark.py

```diff
@@ -356,7 +356,8 @@ def main(dataset_name: str, path: Path, metrics: List[str], batch_size: int, dev
     parser.add_argument('--metrics', nargs='+', default=[], help='Metrics to benchmark',
                         choices=list(METRICS.keys()) + list(METRIC_CATEGORIES.keys()) + ['all'])
     parser.add_argument('--batch_size', type=int, default=1, help='Batch size')
-    parser.add_argument('--device', default='cuda', choices=['cpu', 'cuda'], help='Computation device')
+    device_choice = ['cpu', 'cuda'] + [f'cuda:{i}' for i in range(torch.cuda.device_count())]
```

Comment:
I think that we do not need to test each and every GPU because it is not our functionality. It comes from PyTorch that we rely on. Especially considering that there are no checks in this test that validate that data was sent to correct devices.

Reply:
This change is to simplify user experience rather than testing all available devices. Old configuration doesn't allow use of non-zero device as `cuda:n` is not presented in choices. New dynamic choices helps to use a particular device for benchmarking on desired GPU.